openbb_terminal/portfolio/portfolio_engine.py

```diff
@@ -284,6 +285,19 @@ def __preprocess_transactions(self):
             self.__transactions["Quantity"] = (
                 abs(self.__transactions["Quantity"]) * self.__transactions["Signal"]
             )
+            
+            # Adjust quantity and price for splits 
+            for ticker in self.__transactions["Ticker"].unique() :
+                try :
+                    splits_df = get_splits(ticker)
+                    if not splits_df.empty :
+                        splits_df = splits_df.tz_localize(tz=None)
+                        for split_date in splits_df.index :
+                            self.__transactions['Quantity'] = np.where((self.__transactions['Ticker'] == ticker) & (self.__transactions['Date'] < split_date), self.__transactions['Quantity'] * splits_df.loc[split_date].values , self.__transactions['Quantity'])
+                            self.__transactions['Price'] = np.where((self.__transactions['Ticker'] == ticker) & (self.__transactions['Date'] < split_date), self.__transactions['Price'] / splits_df.loc[split_date].values , self.__transactions['Price'])
```

Comment:
Also isn't this not needed because the price is already split adjusted (which is why we need to adjust quantity in the first place)?  If the value is Price * Quantity and its incorrect, then Price / Split * Quantity * Split is the same and will still be incorrect.

Reply:
yes Im aware that we use Adj. Close. But if you don't adjust the price here in get_transactions one will get quantity that is after split but price that is before split. Hence its introduced here.